strings/example_test.go

```diff
@@ -25,6 +25,24 @@ func ExampleSplit() {
 	// { "a", "b", "c", "d" }
 }
 
+// strings.fields(string)
+func ExampleFields() {
+	state := lua.NewState()
+	Preload(state)
+	inspect.Preload(state)
+	source := `
+	local strings = require("strings")
+	local inspect = require("inspect")
+	local result = strings.fields("a b c d")
+    print(inspect(result, {newline="", indent=""}))
```

Comment:
This appears indented differently from the other lines - I'm on mobile so can't be sure. Can you check?

Reply:
You're right. The indents are mixed tab with space. I've fixed it.